quinn-proto/src/endpoint.rs

```diff
@@ -50,6 +50,8 @@ pub struct Endpoint {
     allow_mtud: bool,
     /// Time at which a stateless reset was most recently sent
     last_stateless_reset: Option<Instant>,
+    /// Buffered 0-rtt messages for pending incoming connections
```

Comment:
These aren't necessarily 0-RTT. They might be Initial packets bearing extra handshake information, or even just padding intended to mitigate anti-amplification limits.

Reply:
Done.